webapp/utils/utils.jsx

```diff
@@ -96,6 +97,15 @@ export function notifyMe(title, body, channel, teamId, duration, silent) {
         notificationDuration = duration;
     }
 
+    // If notification is meant for the current active channel, don't
```

Comment:
You can actually remove this part. `Utils.notifyMe` should only be called from the `NotificationStore` so we're just checking the focus twice.